src/Microsoft.PowerShell.ConsoleHost/host/msh/ManagedEntrance.cs

```diff
@@ -54,34 +76,30 @@ public static int Start(string consoleFilePath, [MarshalAs(UnmanagedType.LPArray
             Thread.CurrentThread.CurrentUICulture = NativeCultureResolver.UICulture;
             Thread.CurrentThread.CurrentCulture = NativeCultureResolver.Culture;
 
-#if DEBUG
-            if (args.Length > 0 && !string.IsNullOrEmpty(args[0]) && args[0].Equals("-isswait", StringComparison.OrdinalIgnoreCase))
-            {
-                Console.WriteLine("Attach the debugger to continue...");
-                while (!System.Diagnostics.Debugger.IsAttached)
-                {
-                    Thread.Sleep(100);
-                }
+            // Warm up some components concurrently on background threads.
+            EarlyStartup.Init();
```

Comment:
Don't move this line. It should be at the very beginning like before.

Reply:
Moved but after debugger wait code.